### PR TITLE
Update ETL logging

### PR DESCRIPTION
**Pull request by aaron-collier**
Update the ETL logging to allow the cloudwatch agent to track a file and upload to cloudwatch.

### CODE REVIEW

File: lib/rialto/etl/logging.rb [hunk 1]
```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
           # Everything to stderr
-          logger.adapter :stderr
+          # logger.adapter :stderr
+          logger.adapter :datefile, 'rialto_etl.log', level: %i[debug info warn]
```
Comment (mjgiarlo) [inline lib/rialto/etl/logging.rb]: We use `:warn` all over the place. :+1: to either using `:error` in all the spots where we actually want to know that happened, or including `:warn` in the error log.

File: lib/rialto/etl/logging.rb [hunk 2]
```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
           # Everything to stderr
-          logger.adapter :stderr
+          # logger.adapter :stderr
```
Comment (mjgiarlo) [inline lib/rialto/etl/logging.rb]: Cruft to be cleaned up later, but I suspect this was already on your to-do list!

File: lib/rialto/etl/logging.rb [hunk 3]
```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
```
Comment (mjgiarlo) [inline lib/rialto/etl/logging.rb]: Given you're already touching a lot of our logging in this branch, I wonder if we can simplify this to:

```ruby
  @logger ||- Yell.new do |logger|
    # ...
  end
```

That is the incantation as described in the Yell README. Does invoking it with `:null` and tapping into it get us something? (Maybe a @justinlittman question.)

File: lib/rialto/etl/writers/sparql_writer.rb [hunk 4]
```diff
@@ -111,11 +113,6 @@ def post(statements)
           raise
         end
 
-        # Get the logger from the settings, or default to an effectively null logger
```
Comment (mjgiarlo) [inline lib/rialto/etl/writers/sparql_writer.rb]: Different approach from the SPARQL Statement Writer?

File: lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb [hunk 5]
```diff
@@ -9,12 +9,15 @@
 extend TrajectPlus::Macros::JSON
 extend Rialto::Etl::NamedGraphs
 extend Rialto::Etl::Vocabs
+extend Rialto::Etl::Logging
```
Comment (mjgiarlo) [inline lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb]: Does this need a `require` statement above? (I didn't mention this earlier.)

File: lib/rialto/etl/logging.rb [hunk 6]
```diff
@@ -10,13 +10,9 @@ module Logging
...
 
       def logger
-        @logger ||= Yell::Logger.new(:null).tap do |logger|
-          # Everything to stderr
-          # logger.adapter :stderr
-          logger.adapter :datefile, 'rialto_etl.log', level: %i[debug info warn]
-          logger.adapter :datefile, 'rialto_etl_err.log', level: %i[error fatal]
-          # Warnings to file
-          # logger.adapter :datefile, 'warnings.log', level: [:warn]
+        @logger ||= Yell::Logger.new do |logger|
```
Comment (mjgiarlo) [inline lib/rialto/etl/logging.rb]: This looks a bunch better! Could we take it one little step further and remove the `::Logger` too, or is that required?

Comment (aaron-collier) [reply lib/rialto/etl/logging.rb]: Nope, removed.

File: lib/rialto/etl/writers/sparql_writer.rb (hunk 4 above)
Comment (aaron-collier) [reply lib/rialto/etl/writers/sparql_writer.rb]: Can you clarify? I don't see any logging in the sparql statement writer.

Comment (mjgiarlo) [reply lib/rialto/etl/writers/sparql_writer.rb]: Outdated comment, methinks.

File: lib/rialto/etl/readers/ndjson_reader.rb [hunk 7]
```diff
@@ -1,21 +1,24 @@
...
 
         def initialize(input_stream, settings)
           @settings = settings
           @input_stream = input_stream
         end
 
-        def logger
-          @logger ||= (@settings[:logger] || Yell.new(STDERR, level: 'gt.fatal')) # null logger)
-        end
+        # def logger
```
Comment (mjgiarlo) [inline lib/rialto/etl/readers/ndjson_reader.rb]: Cruft?

File: lib/rialto/etl/configs/sparql.rb [hunk 8]
```diff
@@ -2,6 +2,11 @@
 
 require 'rialto/etl/readers/sparql_statement_reader'
 require 'rialto/etl/writers/sparql_writer'
+
+extend Rialto::Etl::Logging
```
Comment (mjgiarlo) [inline lib/rialto/etl/configs/sparql.rb]: Does this need a `require` statement above?

Comment (aaron-collier) [reply lib/rialto/etl/configs/sparql.rb]: Added.

**Pull request merged**